### DISCUSSION TITLE
Images are not visible on worker machines in RKE2 cluster

### DISCUSSION THREAD

**Discussion by koraykutanoglu** [rancher/rke2#3734] [Q&A]
Hello, the problem is actually very simple. I'm setting up a cluster with RKE2 infrastructure. Since containerd is used, I'm trying to list images and containers with the following command:

```
ctr container list
```

```
ctr image list
```

But unfortunately the output of these commands is empty. How do I view or image my containers running on worker machines?

**Comment by imageschool** [comment]
Try below? 

```
export PATH="/var/lib/rancher/rke2/bin:$PATH"
sudo env "PATH=$PATH" crictl --config /var/lib/rancher/rke2/agent/etc/crictl.yaml ps
```

**Comment by brandond** [comment]
if you're using `ctr` to talk to containerd directly via its custom GRPC interface, you need to specify a namespace. Kubernetes stores everything under the `k8s.io` namespace instead of `default`, so you should set `--namespace=k8s.io`. If you use `crictl`, I believe this is handled for you as part of the CRI interface.
